python/ray/_private/usage/usage_lib.py

```diff
@@ -62,18 +66,80 @@
 #################
 
 
+@dataclass(init=True)
+class UsageStatsToReport:
+    """Usage stats to report"""
+
+    ray_version: str
+    python_version: str
+    schema_version: str
+    source: str
+    session_id: str
+    git_commit: str
+    os: str
+    collect_timestamp_ms: int
+    session_start_timestamp_ms: int
```

Comment:
Can we also include the sequence number, and number of failed / successful reports? That's useful data, and easy enough to add.